test/library/packages/ArgumentParser/MainWithoutArgumentParser.chpl

```diff
@@ -1,5 +1,6 @@
 
 module M {
+  use Map;
```

Comment:
Why does this need a use of Map?  Was that accidentally committed?

Reply:
This was because the `COMPOPTS` file is now throwing `-smapMethodsThrow=true`, which requires the Map module to be `use`d. Rather than moving this to a new directory, I thought it simpler to just add a `use Map`. Do you disagree with that?

Reply:
Ohhhhh, okay.  I think it would be better to move the test, personally, but I wouldn't hold the PR up over it.